app/models/topical_event.rb

```diff
@@ -97,18 +97,7 @@ def consultations
   end
 
   def announcements
-    announcement_document_types = %w[
-      press_release
-      news_article
-      news_story
-      fatality_notice
-      speech
-      written_statement
-      oral_statement
-      authored_article
-      government_response
-    ]
-    @announcements ||= @documents_service.fetch_related_documents_with_format({ filter_content_store_document_type: announcement_document_types })
+    @announcements ||= @documents_service.fetch_related_documents_with_format({ filter_content_purpose_supergroup: "news_and_communications" })
```

Comment:
It looks like ["news_and_communications"](https://github.com/alphagov/govuk_document_types/blob/main/data/supergroups.yml#L6-L11) will also include [other document types](https://github.com/alphagov/govuk_document_types/blob/main/data/supertypes.yml#L158-L188), not listed in the array that's being replaced. Just wanted to check that it is expected. 

You can use `reject_content_purpose_subgroup` to filter stuff out.

Reply:
Ahh, good spot. I'm not too worried about the original list as it was just copied from Whitehall and I'm inclined not to read too much into the exact list we had there, but have now updated this query to exclude the same content_purpose_supergroups as we're [excluding elsewhere in this app for news_and_communications](https://github.com/alphagov/collections/blob/fc9db4ef6d87ad52d0ac795d1dd05d5fa116dde1/app/services/search/supergroups.rb#L17). We should also switch to using this same supergroups code at some point (don't think we saw it at the time) but I'm considering this outside of the scope of this PR.